src/service/video-manager-impl.js

```diff
@@ -317,14 +317,23 @@ export class VideoManager {
   }
 
   /**
-   * Gets the current analytics details for the given video.
+   * Gets the current analytics details property for the given video.
    * Fails silently if the video is not registered.
-   * @param {!AmpElement} videoElement
+   * @param {string} id
+   * @param {string} property
    * @return {!Promise<!VideoAnalyticsDetailsDef|undefined>}
    */
-  getAnalyticsDetails(videoElement) {
```

Comment:
Replacing this breaks `analyticsEvent` below.

Reply:
Ohhhh these are different object types, nvm :)

Reply:
Yea, it's also confusing that the VideoEntry and the VideoManager have the same method... maybe I should just change that now for clarity?